pages/en/lb2/Using-promises.md

```diff
@@ -72,6 +72,38 @@ Promise support in LoopBack is still in progress. The following are complete:
 
 See [LoopBack issue #418](https://github.com/strongloop/loopback/issues/418#issue-38984704) for details.
 
+{% include tip.html content="
+    
+When returning a Promise instead of a callback the callback function should not be called at all. Calling the callback function inside a Promise return will cause unexpected results, for example Operation Hooks being called twice.  For example:
+```javascript
+MyModel.observe('before save', function beforeSave(ctx, next) {
+  // Resolving with a callback
+  next();
+});
+MyModel.observe('before save', function beforeSave(ctx, next) {
```

Comment:
Since `next` is not used by the function, I personally prefer to leave it from the function signature. That leaves even less temptation to incorrectly call it.

```js
MyModel.observe('before save', function beforeSave(ctx) {
  // Resolving with a promise
  return Promise.resolve();
});
```